### PR TITLE
Add refinedcrafterproxy

### PR DESCRIPTION
**Pull request by StevenDoesStuffs**
#5400 

Please let me know if I've missed anything (I'm not sure if I have to actually add the mod anywhere or if that's all on your side).

### CODE REVIEW

File: config/configswapper/expert/config/refinedcrafterproxy-common.toml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		#Must be a list of integers of length 5 with elements > 0.
+		updateIntervals = [10, 8, 6, 4, 2]
+		#The maximum number of successful crafting updates per crafter update,
+		#where the (zero-indexed) index in the list is the number of speed upgrades.
+		#Must be a list of integers of length 5 with elements > 0.
+		maximumSuccessfulCraftingUpdates = [1, 2, 3, 4, 5]
+
+	[tiers.extrastorage_iron]
+		displayName = "Iron"
+		crafterEnergyUsage = 15
```
Comment (NielsPilgaard) [inline config/configswapper/expert/config/refinedcrafterproxy-common.toml:34]: I'm thinking this should be at least 64, I'm unsure how much the Iron Crafter actually uses though. Same goes for the other tiers, at least 64, unsure how much 😄

**Review by NielsPilgaard — CHANGES_REQUESTED** [review]
This is such a boss move :P Very nice work, you've followed our coding guidelines to a T!

There's just that one comment I'd like addressed, and then I'll happily merge this. There's no release planned for E6 in the immediate future, but I'm sure we'll make one eventually.

File: config/configswapper/expert/config/refinedcrafterproxy-common.toml (hunk 1 above)
Comment (StevenDoesStuffs) [reply config/configswapper/expert/config/refinedcrafterproxy-common.toml:34]: The upgraded crafters use `15 * tier` for their base energy (for 1.16.5 this is actually hardcoded into extrastorage, see [here](https://github.com/Edivad99/ExtraStorage/blob/a628fb87c8084a118c9cc3b064bd6d836c32a184/src/main/java/edivad/extrastorage/nodes/AdvancedCrafterNetworkNode.java#LL165C54-L165C54)). Currently all the settings match their respective crafters. Do you still want me to change this?

Comment (NielsPilgaard) [reply config/configswapper/expert/config/refinedcrafterproxy-common.toml:34]: Ahh okay, I wasn't aware - Then this is good to go 👌

**Review by NielsPilgaard — APPROVED** [review]
Thank you for the mod and integration :D

**Comment by StevenDoesStuffs** [comment]
Done :smile:

**Pull request merged**